content/_changelogs/10.11.0.md

```diff
@@ -0,0 +1,32 @@
+
+- Adds support for [`cy.session()`](/api/commands/session) to experimental WebKit.
+Ensures that with `experimentalSessionAndOrigin` enabled, tests are still able to run against webkit. Addresses [#24116](https://github.com/cypress-io/cypress/issues/24116), [#23832](https://github.com/cypress-io/cypress/issues/23832).
+- Improves user experience when connecting a project to the Dashboard. Addresses [#23379](https://github.com/cypress-io/cypress/issues/23379), [#23766](https://github.com/cypress-io/cypress/issues/23766), [#23767](https://github.com/cypress-io/cypress/issues/23767).
+  - Users can now connect a project to the Dashboard when logging in from the Cypress Launchpad after a testing type has been selected.
+  - Users will be prompted to record runs to the Dashboard from both the Cypress App and Cypress Launchpad if a project is connected to the Dashboard and has no recorded runs.
+
+**Bugfixes:**
+
+- [`cy.origin()`](/api/commands/origin) now supports more than 30 unique origin spec bridges per test. Fixes [#22874](https://github.com/cypress-io/cypress/issues/22874), [#23967](https://github.com/cypress-io/cypress/issues/23967).
+- Fixed an issue where `document.cookie` would not reflect the correct value in cross-origin tests. Fixes [#23531](https://github.com/cypress-io/cypress/issues/23531)
+- Fixed as issue where browser-skipped tests were incorrectly recorded to the Dashboard which resulted in the Dashboard marking the test as "new" or "modified" when it already existed. Fixes [#23517](https://github.com/cypress-io/cypress/issues/23517).
+- Cypress will now restart on changes to the `blockHosts` configuration entry. Fixes [#22634](https://github.com/cypress-io/cypress/issues/22634).
+- When shown the command to record runs to the cloud, the user can select and copy parts of the command (like the record key), instead of only being able to use the "Copy" button to copy the entire command. Fixes [#22091](https://github.com/cypress-io/cypress/issues/22091).
+- Cleaned up inconsistencies in the UI between sentence case and title case. Fixes [#21854](https://github.com/cypress-io/cypress/issues/21854).
+- Fixed an issue where there is a visible "Project ID" section (with no `projectId`) when user has not connected to the Dashboard. [#21806](https://github.com/cypress-io/cypress/issues/21806).
+- When a chromium based browser tab or process crashes, Cypress will no longer hang indefinitely but will fail the current test and move on to the next. [#6170](https://github.com/cypress-io/cypress/issues/6170).
+
+**Experimental Breaking Changes:**
+
+- [`cy.origin()`](/api/commands/origin) now supports using `require()` and dynamic `import()` to include
+  dependencies. `Cypress.require()` has been removed. Addresses
+  [#24293](https://github.com/cypress-io/cypress/issues/24293).
+- Fixed issue connecting to the cloud when a self-signed cert was in the cert chain. Fixes [#24298](https://github.com/cypress-io/cypress/issues/24298).
```

Comment:
This one should be up with the bugfixes.

Reply:
@mschile You highlighted two issues.  Are you talking about the "self-signed cert" issue?

Reply:
I believe GitHub just adds the extra context. I only selected line 29. Sorry for the confusion.

Reply:
Fixed